Makefile

```diff
@@ -40,6 +40,9 @@ CXXGEN   := $(BUILDDIR)/cpp
 PYGEN    := $(BUILDDIR)/python
 
 # generated Documentation files
+# default to asciidoc template
+# for mk-docs formatting, pass in TEMPLATE pointing to the mk-docs template
+TEMPLATE := $(SRCDIRINV)/asciidoc.mustache
```

Comment:
In order to be overwritten from the command line the assignment needs to use `?=` over `:=`